src/marketplace/find-and-install/index.md

```diff
@@ -9,24 +9,40 @@ tags:
   - 'abtnode'
 ---
 
-Your ABT Node Marketplace is your digital catalog of Blocklets from ArcBlock, ecosystem partners and software vendors that make it easy to find, install, manage and one-click deploy Blocklets on your ABT Node.
 
-![](./images/abtnode-overview-2-en.png)
+Your ABT Node Marketplace is your digital catalog of Blocklets from ArcBlock, ecosystem partners, and software vendors that make it easy to find, install, manage and one-click deploy Blocklets on your ABT Node.
```

Comment:
Your ABT Node Marketplace is your => The marketplace is a digital catelog of blocklets fetched from any blocklet registry instance.

The blocklets are created by...

You can do ... in the marketplace.

Reply:
fixed